pkg/accounts/adaptor/repository/dummy.ts

```diff
@@ -47,6 +48,13 @@ export class InMemoryAccountRepository implements AccountRepository {
     return Promise.resolve(Option.some(account));
   }
 
+  findManyByID(id: AccountID[]): Promise<Result.Result<Error, Account[]>> {
+    const accounts = Array.from(this.data).filter((a) =>
+      id.includes(a.getID()),
+    );
```

Comment:
```suggestion
    const set = new Set<AccountID>(id);
    const accounts = Array.from(this.data).filter((a) =>
      set.has(a.getID()),
    );
```
毎回配列を線形探索するとパフォーマンスが少し気になるので, こうしてはどうでしょうか.